src/app/hikes/hikes-routing.module.ts

```diff
@@ -21,7 +25,7 @@ const routes: Routes = [
       },
       {
         path: ':id',
-        component: HikeDetailComponent,
+        component: HikeDetailComponent
```

Comment:
Arrrrrgh! 
Must... have... comma... dangle... or... won't... be... able... to... breathe (gasp)

Reply:
not sure if for real or... LOL

Do you want me to add them there? It's one of those things that should probably be added to an editorconfig b/c I'm not consistent with that

Reply:
This is the first I noticed we were doing it differently. I've been using the AirBnB JS styleguide lint rules for so long, I didn't even check the Angular CLI default lint config.

I guess that is tslint.json?

Looks like it's loosy-goosey by default, with no trailing-comma rule setting.